packages/shorebird_cli/pubspec.yaml

```
@@ -1,6 +1,6 @@
 name: shorebird_cli
 description: Command-line tool to interact with Shorebird's services.
-version: 0.28.1
+version: 1.0.0-rc1
```

```suggestion
version: 1.0.0-rc.1
```

Isn't it technically rc.1? https://semver.org/spec/v2.0.0-rc.1.html